pkg/report/report.go

```diff
@@ -652,6 +652,9 @@ func handleOverrides(original, override []*malcontent.Behavior) []*malcontent.Be
 		if b, exists := behaviorMap[o.Override]; exists {
 			b.RiskLevel = o.RiskLevel
 			b.RiskScore = o.RiskScore
+
+			// Delete the override rule from the behavior map
+			delete(behaviorMap, o.RuleName)
```

Comment:
I had to go back and forth on this. Without this we'd modify the original rule and also display the override rule which we don't want.